app/classifier/tasks/dropdown/index.jsx

```diff
@@ -0,0 +1,292 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Select from 'react-select';
+import GenericTask from '../generic';
+import DropdownEditor from './editor';
+
+export const DropdownSummary = (props) => {
+  const getOptionsKeys = () => {
+    const optionsKeys = {};
+    props.annotation.value.forEach((answer, i) => {
+      const { id, condition } = props.task.selects[i];
+      if (i === 0) {
+        optionsKeys[id] = answer.value;
+      } else if (answer.value === null) {
+        optionsKeys[id] = null;
+      } else {
+        optionsKeys[id] = `${optionsKeys[condition]};${answer.value}`;
+      }
+    });
+    return optionsKeys;
+  };
+
+  const getOptionLabel = (value, i) => {
```

Comment:
This should be declared as a named function too.

I wonder if we can catch these with a linter rule?